doc/TUTORIAL.md

```diff
@@ -96,32 +93,35 @@ Each service has a directory for its files:
 All service badge classes inherit from [BaseService] or another class which extends it.
 Other classes implement useful behavior on top of [BaseService].
 
-* [BaseJsonService](https://github.com/badges/shields/blob/master/services/base-json.js)
+- [BaseJsonService](https://github.com/badges/shields/blob/master/services/base-json.js)
   implements methods for performing requests to a JSON API and schema validation.
-* [BaseXmlService](https://github.com/badges/shields/blob/master/services/base-xml.js)
+- [BaseXmlService](https://github.com/badges/shields/blob/master/services/base-xml.js)
   implements methods for performing requests to an XML API and schema validation.
-* If you are contributing to a *service family*, you may define a common super
+- If you are contributing to a _service family_, you may define a common super
   class for the badges or one may already exist.
 
-[BaseService]: https://github.com/badges/shields/blob/master/services/base.js
+[baseservice]: https://github.com/badges/shields/blob/master/services/base.js
```

Comment:
should be `https://github.com/badges/shields/blob/master/core/base-service/base.js`